common/models/transducer/alignment.py

```diff
@@ -64,3 +66,35 @@ def rnnt_alignment(source, **kwargs):
   _, alignment = tf_forward_shifted_rnnt(log_probs, targets.get_placeholder_as_batch_major(), enc_lens, dec_lens,
                                          blank_index=blank_idx, debug=False, with_alignment=True)
   return alignment  # (B, T)
+
+
+def rna_alignment_out_type(sources, **_kwargs) -> Data:
+  """ Computes the rna-alignment Data_out_type for RNA alignment
+  B: batch, T: time, U:target/labels, V: vocabulary
+  sources: [output_log_prob, real_target, "base:encoder"]
+  Args:
+      sources ([LayerBase, LayerBase, LayerBase]): [output_log_prob, real_target, base:encoder]
```

Comment:
You repeat yourself here three times what `sources` is. Just say it once, not three times.
And actually the type of `sources` is missing here (`list[LayerBase]`).